test/marketplace/MarketPlace.sol

```diff
@@ -64,12 +63,13 @@ contract MarketPlace {
     string memory n,
     string memory s,
     uint8 d
-  ) public onlyAdmin(admin) returns (bool) {
-    require(swivel != address(0), 'swivel contract address not set');
+  ) public onlyAddress(admin) returns (bool) {
```

Comment:
`public authorized(admin)`... we can normalize the contracts to use that

Reply:
👍